app/scripts/background.js

```diff
@@ -383,7 +383,9 @@ function setupController(initState, initLangCode, remoteSourcePort) {
         // log error so we dont break the pipeline
         if (!dataPersistenceFailing) {
           dataPersistenceFailing = true;
-          captureException(err);
+          if (state.data.metamask.participateInMetaMetrics) {
```

Comment:
I don't think we need this _and_ the modifications to `beforeSend`. The `beforeSend` changes should be adequate on their own, and it avoids having to be vigilant about `captureException` calls.

Reply:
The same goes for the `captureException` wrapper added in `metamask-controller.js`

Reply:
but this `captureException` is imported directly `import { captureException } from '@sentry/browser';`, it's not from `global.sentry`, so I didn't think the `beforeSend` change would apply to it.

Reply:
I guess we should replace all cases of `import { captureException } from '@sentry/browser';` with `const { captureException } = global.sentry`?

Reply:
The Sentry library acts as a singleton. It must be initialized once with `.init`, and those settings take effect globally. Without that `init` call, it doesn't work at all.

Reply:
Okay, I've remove this in d7202b4e0